src/endpoints/borrowed/put/return_borrowed_for_user.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from fastapi import Depends, HTTPException, Path, status
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+
+from src.dependencies import get_current_user, get_db
+from src.endpoints.borrowed.router_init import router
+from src.models import all_models
+from src.schemas.borrowed import BorrowedSchema
+from src.responses import custom_response
+
+
+@router.put("/return_borrowed_user/{borrowed_id}", response_model=None, status_code=status.HTTP_200_OK)
+async def return_borrowed_for_user( borrowed: BorrowedSchema,
+    borrowed_id: int = Path(gt=-1),
+    db: Session = Depends(get_db),
+    user: dict = Depends(get_current_user)) -> dict:
+    """
+    This function will be used to update a borrowed by id.
+    Parameters:
+        borrowed_id: The id of the borrowed.
+        borrowed: The borrowed data.
+        user: The user data. (current user)
+        db: The database session.
+    Returns:
+        A dict that contains the status_code, detail and the data.
+    """
+    logging.info("Updating borrowed in database with id: " + str(borrowed_id))
+    found_borrowed = db.scalar(
+        select(all_models.Borrowed).where(all_models.Borrowed.id == borrowed_id)
```

Comment:
There should be another where condition to check if the user_id is the same as logged in user's id.